components/gitpod-db/src/typeorm/migration/1679410564277-AdminUserDropAdminRole.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { BUILTIN_INSTLLATION_ADMIN_USER_ID } from "../../user-db";
+
+export class AdminUserDropAdminRole1679410564277 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(
+            `UPDATE d_b_user SET rolesOrPermissions = '[]' WHERE id = '${BUILTIN_INSTLLATION_ADMIN_USER_ID}'`,
```

Comment:
The implication here is if this version gets deployed to an env, which has an existing admin (or is about to create the "initial user admin", they will not be able to access admin. This may be fine, but just calling it out here.

Reply:
> they will not be able to access admin

To make sure we're on the same page: "they will not be able to access `/admin`": Yes. :white_check_mark:
The goal of this change is to deny user's access to the installation-level admin dashboard. :+1: 

:cloud: Further thoughts:
 - So far, the only requirements for this user are "enable onboarding"
 - `/admin` duties should move into orgs anyways (as much as possible)
 - If we need some (synthetic) user to access `/admin` again (to enable Gitpod employees to access it? hypothetic use case atm!), we can come up with a new one/reconsider this choice.